xml/rmt_install.xml

```diff
@@ -77,14 +77,64 @@
   </para>
  </sect1>
  <sect1 xml:id="sec-rmt-installation-zypper">
-  <title>Installation on existing system</title>
+  <title>Installation on an existing system</title>
 
   <para>
    To install &rmt; on a running &productname; installation, use
    <command>zypper</command>:
   </para>
 
 <screen>&prompt.sudo;<command>zypper in rmt-server</command></screen>
+<sect2 xml:id="sec-rmt-installation-zypper-jeos">
+ <title>Installation on &jeos;</title>
+ <para>
+  &sls; &jeos; (Just enough Operating System) is a minimal customizable
+  operating system that is designed for a specific usage scenarios, for
+  example to be run as:
+ </para>
+ <itemizedlist>
+  <listitem>
+   <para>
+    A container host.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    A virtual machine guest.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    An appliance base system.
+   </para>
+  </listitem>
+  <listitem>
+   <para>
+    A small server image.
+   </para>
+  </listitem>
+ </itemizedlist>
+ <para>
+  Therefore &jeos; image is a good choice for being used as an &rmt; server.
+  You can download &sls; &jeos; images for &kvm;, &xen;, &hyperv;, &vmware;,
+  and OpenStack from the public &sls; download page at <link
+   xlink:href="https://www.suse.com/download/sles/"/>. Find more information
+  on &jeos; at <link
+   xlink:href="https://documentation.suse.com/sles/15-SP4/html/SLES-all/article-jeos.html"/>.
+ </para>
+ <para>
+  Installation of &rmt; on &jeos; works identical to installing it on an
+  already installed system (see <xref linkend="sec-rmt-installation-zypper"/>.
```

Comment:
There seems to be a font discrepancy for the following part: 'Repository Mirroring Tool Guide' which is not italicized like the rest of the sentence:
(see Book “Repository Mirroring Tool Guide”, Chapter 2 “RMT installation and configuration”, Section 2.3
“Installation on an existing system”).

<img width="501" alt="jeos" src="https://user-images.githubusercontent.com/98878014/157062497-7869c2c8-0ebc-4660-872f-e6b09cabfc41.png">

Reply:
it's not in italics because it is not a link probably, we cannot affect this in the source code, it's a matter of rendering